src/Sylius/Bundle/CurrencyBundle/Templating/Helper/CurrencyHelper.php

```diff
@@ -74,12 +74,23 @@ public function convertAmount($amount, $currency = null)
     /**
      * {@inheritdoc}
      */
+    public function formatAmount($amount, $currency, $locale = null)
+    {
+        if($locale) {
```

Comment:
Missing space after ``if``. I would also modify check to ``(null !== $locale)``.